PointOfDispensingApp/src/components/VaccinationScreeningComponent.vue

```diff
@@ -1,95 +1,284 @@
+            <v-radio label="Yes" value="Yes"></v-radio>
+            <v-radio label="No" value="No"></v-radio>
+          </v-radio-group>
+        </v-row>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-divider></v-divider>
+    </v-row>
+    <v-row>
+      <v-col cols="10">
+        <v-row>
+          <v-radio-group>
+            <template v-slot:label>
+              <div><span class="red--text"><strong>* </strong></span>
+              <span class="font-weight-medium secondary--text">6. Have you received any vaccine in the last 14 days?</span></div>
+            </template>
+          </v-radio-group>
+        </v-row>
+      </v-col>
+      <v-col cols="2">
+        <v-row align="center" justify="end">
+          <v-radio-group v-model="screeningQ6" row @change="screeningChecklistUpdate()">
+            <v-radio label="Yes" value="Yes"></v-radio>
+            <v-radio label="No" value="No"></v-radio>
+          </v-radio-group>
+        </v-row>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-divider></v-divider>
+    </v-row>
+    <v-row>
+      <v-col cols="10">
+        <v-row>
+          <v-radio-group>
+            <template v-slot:label>
+              <div><span class="red--text"><strong>* </strong></span>
+              <span class="font-weight-medium secondary--text">7. Are you currently under isolation or quarantine due to COVID-19?</span></div>
+            </template>
+          </v-radio-group>
+        </v-row>
+      </v-col>
+      <v-col cols="2">
+        <v-row align="center" justify="end">
+          <v-radio-group v-model="screeningQ7" row @change="screeningChecklistUpdate()">
+            <v-radio label="Yes" value="Yes"></v-radio>
+            <v-radio label="No" value="No"></v-radio>
+          </v-radio-group>
+        </v-row>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-divider></v-divider>
+    </v-row>
+    <v-row>
+      <v-col cols="10">
+        <v-row>
+          <v-radio-group>
+            <template v-slot:label>
+              <div><span class="red--text"><strong>* </strong></span>
+              <span class="font-weight-medium secondary--text">8. Have you received passive antibody therapy or convalescent plasma as treatment for COVID-19 in the past 90 days?</span></div>
+            </template>
+          </v-radio-group>
+        </v-row>
+      </v-col>
+      <v-col cols="2">
+        <v-row align="center" justify="end">
+          <v-radio-group v-model="screeningQ8" row @change="screeningChecklistUpdate()">
+            <v-radio label="Yes" value="Yes"></v-radio>
+            <v-radio label="No" value="No"></v-radio>
+          </v-radio-group>
+        </v-row>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-divider></v-divider>
+    </v-row>
+      <v-col cols="12">
+        <v-card class="mx-auto my-12" color="accent" dark height="50%" v-show="screeningComplete">
+          <v-card-title class="headline justify-center">
+              Proceed with vaccination?
```

Comment:
It's a jarring UX where the user selecting yes/no on this button doesn't have some indication to go to a new page (or auto moving to that page). 

This could prob be a new card.

Reply:
If this is evaluated during the check-in process (as Teri's notes from WLFD imply), then auto-advancing to the Vaccination Event page would not be appropriate.

The good news is that the workflow tweaks you've suggested in this batch of cards are all very doable.  I think we just need to learn a little more before we decide what makes the most sense.